lib/query-interface.js

```diff
@@ -406,7 +406,7 @@ QueryInterface.prototype.renameColumn = function(tableName, attrNameBefore, attr
 
     _options[attrNameAfter] = {
       attribute: attrNameAfter,
-      type: data.type,
+      type: DataTypes[data.type],
```

Comment:
@fixe  This was breaking a mysql test so I removed it. Tests seem to pass fine without this change